src/main/java/com/kuljava/swiatwsi/rawmaterials/RawMaterialsRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.kuljava.swiatwsi.rawmaterials;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+
+@Repository
+public interface RawMaterialsRepository extends JpaRepository<RawMaterials, Long> {
+
+    @Transactional
+    @Modifying
+    @Query("UPDATE RawMaterials r SET r.woodQuantity = r.woodQuantity + ?1, r.clayQuantity = r.clayQuantity + ?2, r.ironQuantity = r.ironQuantity + ?3")
+    void increaseAll(Long woodIncr, Long clayIncr, Long ironIncr);
```

Comment:
Just for the future. We should want to minimize the amount of parameters. Method without parameters is ideal, one parameter is good, two parameters are acceptable. If we have more parameters, think about wrapping them into a class (probably there exists one like RawMaterials).

Reply:
Ok,but are you sure that  it is neccessary here? Is it not unnecessary creation of entities?

Reply:
If You delete this method, You won't need it. It is not a comment about this particular method, but just for the future